### PR TITLE
Add component documentation for media_player/anthemav

### PR DESCRIPTION
**Pull request by nugget**
**Description:**

Add component page for new platform media_player/anthemav

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#5146

### CODE REVIEW

File: source/_components/media_player.anthemav.markdown [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: anthemav
+    host: IP_ADDRESS
+    port: 14999
+```
+
+### Supported Options
```
Comment (armills) [inline source/_components/media_player.anthemav.markdown]: Can you update the list of options to match the format of other pages? [One example here](https://github.com/home-assistant/home-assistant.github.io/blob/current/source/_components/media_player.kodi.markdown).

File: source/_components/media_player.anthemav.markdown [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Item Name       | Type | Required?   | Description
+----------------|------|-------------|-------------------------------------------
+`host`          | str  | *required*  | Hostname or IP Address of device
+`port`          | int  |  optional   | TCP port number of device
```
Comment (armills) [inline source/_components/media_player.anthemav.markdown]: Include the default port.

File: source/_components/media_player.anthemav.markdown [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+[anthemav]: https://github.com/nugget/python-anthemav
+
+
+## {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: anthemav
+    host: IP_ADDRESS
```
Comment (armills) [inline source/_components/media_player.anthemav.markdown]: Show an actual IP address here as an example.

File: source/_components/media_player.anthemav.markdown [hunk 4]
```diff
@@ -36,17 +36,15 @@ currently supported.
...
 
-### Supported Options
+### Configuration Variables
 
-Item Name       | Type | Required?   | Description
-----------------|------|-------------|-------------------------------------------
-`host`          | str  | *required*  | Hostname or IP Address of device
-`port`          | int  |  optional   | TCP port number of device
-`name`          | str  |  optional   | Defaults to model name if not provided
+- **host** (*Required*): The host name or address of the device that is running XBMC/Kodi
```
Comment (armills) [inline source/_components/media_player.anthemav.markdown]: Looks good, except it says XBMC/Kodi. :)

Comment (nugget) [inline source/_components/media_player.anthemav.markdown]: gah!  How embarassing

**Pull request merged**